cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,16 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => cy.visit('https://conduit.mate.academy/user/login'));
+
   it('should provide an ability to log in', () => {
+    cy.url().should('contain', 'user/login');
+
+    cy.get(':nth-child(1) > .form-control').type('fortest123123@gmail.com');
+    cy.get(':nth-child(2) > .form-control').type('Qwerty123.');
+
+    cy.get('.btn').click();
 
+    cy.get(':nth-child(4) > .nav-link').should('contain', 'fortest123123');
```

Comment:
The assertion to check the username should be more specific to the user's display area. Using `:nth-child` is not reliable as the page structure can change. It would be better to use an attribute selector or test id like `data-cy`.